Content.Server/Silicons/Borgs/BorgSystem.cs

```diff
@@ -207,10 +214,21 @@ protected override void OnInserted(EntityUid uid, BorgChassisComponent component
     {
         base.OnInserted(uid, component, args);
 
+        string? pAIName = null;
+        /* Goobstation
+           Save the name of the pAI.*/
+        if (HasComp<PAIComponent>(args.Entity))
+            pAIName = $" ({Name(args.Entity)})";
```

Comment:
```suggestion
            pAIName = $"({Name(args.Entity)})";
```
Put the space in string in line 231 and 338 instead.

Reply:
thankk you